dbt/models/hypercert_claims.sql

```diff
@@ -0,0 +1,16 @@
+with source as (
+    select * from {{ source('public', 'raw_hypercert_claims') }}
+),
+
+renamed as (
+    select
+        id,
+        tokenID as token_id,
+        lower(contract) as contract,
+        uri as uri,
```

Comment:
Not 100% sure of the format but seems naming it `ipfs_cid` might be more correct and make things clearer.

What do you think?